src/main/java/nl/tudelft/contextproject/model/PhysicsObject.java

```diff
@@ -12,8 +12,9 @@
 	 * Get spatial object with physics control.
 	 * Each Collidable object has to have some kind of spatial object that 
 	 * implements the PhysicsControl interface.
+	 *
 	 * @return
-	 * 		an object that implements PhysicsControl.
+	 * 		an object that implements PhysicsControl
 	 */
-	public PhysicsControl getPhysicsObject();
+	PhysicsControl getPhysicsObject();
```

Comment:
I see you have installed the tool that jurgen showed? We don't have the CheckStyle rule that checks for "unneccesary modifiers" enabled.

While this change is certainly correct, I do think that explicitly adding the modifier is not necessarily a bad thing. For example, Java 9 will add private methods to interfaces, which would make "no modifier = public" less obvious for interfaces.

Nevertheless, I'm just nitpicking about something that is a personal preference. It seems like it's common practice to leave the modifiers out, so I agree with this change.